### PR TITLE
Creates a script to geocode the IP addresses in form_geos.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/cron/form_geos [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/db'
+require 'cdo/geocoder'
+
+BATCH_SIZE = 1000
+
+def main
+  time_now = DateTime.now
+  to_geolocate = PEGASUS_DB[:form_geos].where(indexed_at: nil).limit(1000)
```
Comment (Hamms) [inline bin/cron/form_geos]: given that this table is currently unpopulated, this limit seems counterproductive. It should at the very least probably be using the `BATCH_SIZE` constant

File: bin/cron/form_geos [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  to_geolocate.each do |form_geo|
+    location = Geocoder.search(form_geo[:ip_address]).first
+    PEGASUS_DB[:form_geos].where(id: form_geo[:id]).update(
+      updated_at: time_now,
+      indexed_at: time_now,
+      city: location.nil? || location.city.blank? ? nil : location.city,
```
Comment (Hamms) [inline bin/cron/form_geos:21]: I believe `nil? || blank?` is redundant

**Comment by Hamms** [comment]
couple minor comments, otherwise LGTM

File: bin/cron/form_geos (hunk 2 above)
Comment (ashercodeorg) [reply bin/cron/form_geos:21]: Nope, noting that `nil?` is applied to `location` and `blank?` is applied to `location.city`.

File: bin/cron/form_geos (hunk 1 above)
Comment (ashercodeorg) [reply bin/cron/form_geos]: Yep, I'll update to use `BATCH_SIZE`. The limit is important, as once we backfill for all existing `forms`, we want this to be throttled.

File: bin/cron/form_geos (hunk 2 above)
Comment (ashercodeorg) [reply bin/cron/form_geos:21]: Note that both of these are cases (`location.nil?` and `location.city.blank?`) happen in production.

Comment (Hamms) [reply bin/cron/form_geos:21]: d'oh.

**Pull request merged**